src/accelerate/test_utils/testing.py

```diff
@@ -29,6 +29,7 @@
 import torch
 
 import accelerate
+from accelerate.utils.imports import is_torchdata_stateful_dataloader_available
```

Comment:
We should add` is_torchdata_stateful_dataloader_available` to `accelerate.utils.__init__.py` and import from there please :)

Reply:
Done